rokka/client.go

```diff
@@ -66,9 +83,32 @@ func (c *Client) Call(req *http.Request, v interface{}) error {
 
 	decoder := json.NewDecoder(resp.Body)
 
-	return decoder.Decode(&v)
+	if resp.StatusCode < 400 {
+		return decoder.Decode(&v)
+	}
+
+	errorBody := ErrorResponse{}
+	err = decoder.Decode(&errorBody)
+	if err != nil {
+		return err
+	}
```

Comment:
the above `if resp.StatusCode < 400` might mean that a 3xx status code could lead to an error here becauase a 3xx code most likely doesn't have an error response body.

Reply:
Indeed, but the potential error is returned here. I agree however, that these cases might need some specific handling, but then again, will this ever happen?

Reply:
the 3xx response we might need to handle separately anyway. Not sure if it happens, really TBH..